### ISSUE TITLE
Changing CWD by adding grunt.file.setBase() results in missing .jshintrc

### ISSUE THREAD

**Issue by rekosko** [micromata/generator-bootstrap-kickstart#31]
Hello,

I've got a small problem after I moved `Gruntfile.js` up one folder from working directory and added `grunt.file.setBase('./wwwroot');` just before registering my tasks. Now if I don't add `.jshintrc` in folder where `Gruntfile.js` is placed and in its working directory in this case `'./wwwroot'` subfolder I'm getting missing file error. So the case is I need 2x `.jshintrc` file... did I miss something? I'm rather new to this stuff so maybe I don't see anything weird in this. 

Your help would be appreciated.

**Comment by mischah** [comment]
Hi,

before I try to reproduce your error I like to introduce you to the workflow we use.
Because it’s possible that you trying to misuse the concept we are providing according to your secription and the fact that you are rather new to this stuff.

The most important: You generate with Grunt a »build« to to the `dist`directory which contain your production ready sources (minified, concatened) wothout the clutter of .dotfiles etc.

The best way to dig into the Grunt workflow is to fire `grunt tasks` after yeoman finished scaffolding and installing the dependencies.  See detailed information in the [readme of the static version of Bootstrap Kickstart](https://github.com/micromata/bootstrap-kickstart#grunt-workflow-and-tasks).

Let my know if you have any other questions after reading and exploring the docs.

Cheers, Michael

**Comment by rekosko** [comment]
Hello,

Thanks for your answer but I found out what I've done wrong.

Basically I've moved the `Gruntfile.js` to different location (one folder up from my assets folder and html files) then I've changed the cwd for grunt. He had trouble to find my `.jshintrc` and what I did is that I've changed the path on https://github.com/micromata/generator-bootstrap-kickstart/blob/master/app/templates/_Gruntfile.js#L83 line from `jshintrc: '.jshintrc',` to `jshintrc: '../.jshintrc',`

Thank you!
